middleware/http/server_test.go

```diff
@@ -101,3 +106,36 @@ func TestHTTPHandlerWrapping(t *testing.T) {
 		t.Errorf("Expected body value %q, got %q", want, have)
 	}
 }
+
+func TestHTTPMethodAsSpanName(t *testing.T) {
```

Comment:
what part of this implies the span name is the method (or is it that method is default?) If the latter, maybe name test to say defaultmethodisspanname?

Reply:
If Span name is not provided as functional option indeed the HTTP Method will be used.

Don't like `defaultmethodisspanname` either.

How about: TestHTTPDefaultSpanName?